google-cloud-logging/lib/google/cloud/logging/async_writer.rb

```diff
@@ -54,58 +55,30 @@ module Logging
       #                       labels: labels
       #
       class AsyncWriter
-        include Stackdriver::Core::AsyncActor
-
-        DEFAULT_MAX_QUEUE_SIZE = 10000
```

Comment:
Are these constants considered part of the public API? Do we need to continue to define them?